src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -0,0 +1,594 @@
+              <TextField
+                id='firstName'
+                label='First name'
+                isRequired
+                mr={{ md: 12 }}
+                mb={{ base: 8, md: 0 }}
+              />
+
+              <TextField id='lastName' label='Last name' isRequired />
+            </Flex>
+
+            {!errors?.firstName && !errors?.lastName && (
+              <PageText as='small' fontSize='helpText' color='brand.helpText'>
+                Who is the point of contact for the application?
+              </PageText>
+            )}
+          </Flex>
+
+          <TextField id='email' label='Email' isRequired />
+
+          <Controller
+            name='POCisAuthorisedSignatory'
+            control={control}
+            render={({ field: { onChange, value } }) => (
+              <Field
+                id='POCisAuthorisedSignatory'
+                label='Is the point of contact also the authorised signatory?'
+                isRequired
+              >
+                <RadioGroup
+                  id='POCisAuthorisedSignatory'
+                  onChange={value => onChange(value === 'Yes')}
+                  value={value ? 'Yes' : 'No'}
+                  fontSize='input'
+                  colorScheme='white'
+                  mt={4}
+                >
+                  <Stack direction='row'>
+                    <Radio
+                      id='POCisAuthorisedSignatory-yes'
+                      size='lg'
+                      name='POCisAuthorisedSignatory'
+                      value='Yes'
+                      defaultChecked
+                      mr={8}
+                    >
+                      <PageText fontSize='input'>Yes</PageText>
+                    </Radio>
+
+                    <Radio
+                      id='POCisAuthorisedSignatory-no'
+                      size='lg'
+                      name='POCisAuthorisedSignatory'
+                      value='No'
+                    >
+                      <PageText fontSize='input'>No</PageText>
+                    </Radio>
+                  </Stack>
+                </RadioGroup>
+              </Field>
+            )}
+          />
+
+          <Box display={notAuthorisedSignatory ? 'block' : 'none'}>
+            <Fade in={notAuthorisedSignatory} delay={0.25}>
+              <TextField
+                id='authorisedSignatoryInformation'
+                label='Name, job title, and email address of the authorised signatory'
+                helpText='(e.g. John Smith, CEO, john@mycompany.com. This is the person who will sign the contract. They must be someone who can sign contracts on behalf of the entity)'
+                isRequired
+              />
+            </Fade>
+          </Box>
+
+          <Controller
+            name='applyingAs'
+            control={control}
+            render={({ field: { onChange } }) => (
+              <Field id='applyingAs' label='In which capacity are you applying?' isRequired>
```

Comment:
Select missing the error PageText when no option is selected

Reply:
Good catch! Fixed! thanks